### DISCUSSION TITLE
ImageMagick silent upgrade on Windows

### DISCUSSION THREAD

**Discussion by gioxx** [ImageMagick/ImageMagick#7370] [General] [answered]
Hello everyone.
I'm trying to silently install an ImageMagick upgrade (we are using version 7.1.1-19 and I would like to upgrade to -33) with the usual command line:

`/VERYSILENT /NORESTART /FORCECLOSEAPPLICATIONS /LOADINF="customsetup.txt"`

Within the file `customsetup.txt` I have these details (saved automatically by the `/SAVEINF` command-line parameter):

```
[Setup]
Lang=default
NoIcons=0
Tasks=modifypath,install_ffmpeg,legacy_support
```

But, when I run the command line, I get the popup below asking if it can overwrite the files in the destination folder. I would like to confirm the operation and allow the upgrade to complete silently, but so far I have not succeeded despite several attempts.

![immagine](https://github.com/ImageMagick/ImageMagick/assets/1768601/f4c64bde-cadb-4295-b6d0-82dbb77f6f1f)

Would anyone be able to help me?
Thanks in advance 😃

**Comment by snibgo** [comment, marked as answer]
Have you tried `/SUPPRESSMSGBOXES`?

**Reply by gioxx** [reply]
Bingo. I have never tried this in my attempts, but this is the solution that works like a charm.
Many thanks for the suggestion.
